build/docker/copy-schema-jar.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+###############################################################################
+# (C) Copyright IBM Corp. 2016, 2020
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################################
+set -e
+
+if [[ -z "${WORKSPACE}" ]]; then
+    echo "ERROR: WORKSPACE environment variable not set!"
+    exit 2
```

Comment:
interesting.. this is considered a reserved exit code
http://www.tldp.org/LDP/abs/html/exitcodes.html

just something to think about next time we're in the code

Reply:
i was wondering why it exits with a code of 2!
lets just change to 1 and be done?

Reply:
```suggestion
    exit 1
```

Reply:
done